example-with-css.css

```diff
@@ -0,0 +1,53 @@
+.cru-recommendations-component {
+  font-family: Source Sans Pro, Sans-Serif;
+  font-weight: bold;
+  color: #383f43;
+  width: 217px;
+}
+.cru-recommendations-component a,
+.cru-recommendations-component a:active,
+.cru-recommendations-component a:visited {
+  color: #383f43;
+  text-decoration: none;
+}
+.cru-recommendations-component h1,
+.cru-recommendations-component h2,
+.cru-recommendations-component h3 {
+  margin: 0;
+}
+.cru-recommendations-component .cru-recommendations-component-header {
+  font-size: 24px;
+  margin-bottom: 32px;
+}
+.cru-recommendations-component .cru-recommendations-component-header::before {
+  content: '';
+  display: block;
+  width: 38px;
+  height: 2px;
+  background: #ffcf07;
+  margin-bottom: 15px;
+}
+.cru-recommendations-component .cru-recommendations-component-card {
+  display: block;
+  margin-bottom: 24px;
+}
+.cru-recommendations-component .cru-recommendations-component-image {
+  display: block;
+  width: 100%;
+  border-radius: 4px;
+  margin-bottom: 12px;
+}
+.cru-recommendations-component .cru-recommendations-component-category {
+  font-size: 14px;
+  letter-spacing: 0.12em;
+  text-transform: uppercase;
+  color: #ffcf07;
+}
+.cru-recommendations-component .cru-recommendations-component-title {
+  font-size: 18px;
+}
+.cru-recommendations-component
+  .cru-recommendations-component-card:hover
+  .cru-recommendations-component-title {
+  color: #5a5a5a;
```

Comment:
@OzzieOrca This isn't a branding color anymore and I haven't gotten direction from Eric about any color variations for hover effects, so let's swap this color change for `text-decoration: underline;`.

Reply:
I just picked a random color from Cru.org. I talked with Ivan after and he wants to do something like https://www.invisionapp.com/inside-design/ has with zooming images and text underlines of the same color (idk how important the animated underline is but sounds optional). I'll work on better hover states in a new PR.

Reply:
@OzzieOrca That animated underline is a really creative use of background sizes. And that #5a5a5a color used to be the go-to dark gray tone, but it got replaced with #383F43 which you're using elsewhere.